### DISCUSSION TITLE
Not enough disk space

### DISCUSSION THREAD

**Discussion by henschelm** [Whisky-App/Whisky#771] [Q&A]
I am running Whisky 2.2.2 on MacOS 14.2.1 (23C71). In my settings it says "1,05 TB available of 2 TB". Now I want install Anno1800 either via steam or directly through UBIConnect. But both tell me that I don't have enough disk space. "Disk Space Available: 41500MB"

Is there a way to increase the Diskspace of the "bottle"?

**Comment by henschelm** [comment]
I found the issue. I had several local snapshots on my system. Although the system setting told me that I had 1,05 TB available of 2 TB, the Disk Utility indicates only 40GB of 2 TB available. So what I did is to remove all the local snapshots with 
1. tmutil listlocalsnapshots /
2. tmutil deletelocalsnapshots 2024-01-10-010907
3. and so on until every snapshot was deleted

After that Disk Utility displays the right amount of free space.

Hope that helps anyone who runs into the same issue.

**Reply by TinyWorkshopDesign** [reply]
same here
